mypyc/test-data/run-integers.test

```diff
@@ -572,3 +572,21 @@ class subc(int):
 [file userdefinedint.py]
 class int:
     pass
+
+[case testBitLength]
+def bit_length(n: int) -> int:
+    return n.bit_length()
+def bit_length_python(n: int) -> int:
+    return getattr(n, "bit_length")()
+def test_bit_length() -> None:
+    assert bit_length(0) == bit_length_python(0)
+    assert bit_length(1) == bit_length_python(1)
+    assert bit_length(255) == bit_length_python(255)
+    assert bit_length(256) == bit_length_python(256)
+    assert bit_length(-256) == bit_length_python(-256)
+    # Large positive int
+    assert bit_length(1 << 70) == bit_length_python(1 << 70)
+    # Large negative int
+    assert bit_length(-(1 << 70)) == bit_length_python(-(1 << 70))
+    # Large int with all bits set
+    assert bit_length((1 << 100) - 1) == bit_length_python((1 << 100) - 1)
```

Comment:
Just in case, can you add something like this (which would replace most of these tests):
```py
    for n in range(1, 256):
        assert bit_length(1 << n) == bit_length_python(1 << n)
        assert bit_length(-(1 << n)) == bit_length_python(-(1 << n))
```
Also check some values which have some or all lower bits set.